sorbet/rbi/shims/activestorage.rbi

```diff
@@ -0,0 +1,9 @@
+# typed: strict
+
+# ActiveStorage dynamically adds a mixin and class attributes
+# so replicating that behaviour here.
+
+class ActiveRecord::Base
+  include(::ActiveStorage::Attached::Model)
+  class_attribute :attachment_reflections, instance_writer: false, default: {}
+end
```

Comment:
Since this happens dynamically, we really should not encode this as if it is statically true. We really should not need this if we operate over the `ActiveStorage::Reflection::ActiveRecordExtensions::ClassMethods` type anyway.